app/src/main/kotlin/com/exallium/todoapp/editnote/EditNotePresenter.kt

```diff
@@ -66,7 +80,27 @@ class EditNotePresenter(view: EditNoteView,
         }).addToComposite()
     }
 
-    fun setupSaveNoteSubscription(noteId: String) {
+    fun setupSaveNewNoteSubscription() {
+        view.saveNoteClicks()
+                .flatMap { model.saveNote(buildNote()).toObservable() }
+                .subscribe(object : Subscriber<Unit>() {
```

Comment:
we can utilize the 2 lambda version of subscribe, where the first lambda is the onNext function, and the second lambda is the onErrror function.